README.md

```diff
@@ -175,7 +175,8 @@ var storage = multer.diskStorage({
     cb(null, '/tmp/my-uploads')
   },
   filename: function (req, file, cb) {
-    cb(null, file.fieldname + '-' + Date.now())
+    const uniqueHash = Date.now() + '-' + Math.random().split('.')[1]
```

Comment:
Can we call this something other than "hash" since it isn't a [hash](https://en.wikipedia.org/wiki/Hash_function)?

Reply:
`Math.random().split('.')[1]` this will throw an error:

> TypeError: Math.random(...).split is not a function